pkg/chain/ethereum/beacon.go

```diff
@@ -451,22 +451,18 @@ func (mrb *mockRandomBeacon) OnDKGStarted(
 				// Generate an event every 500th block.
 				if block%500 == 0 {
 					mrb.currentDkgMutex.Lock()
-
-					if mrb.currentDkgStartBlock == nil {
```

Comment:
We have this check when submitting DKG result:
https://github.com/keep-network/keep-core/blob/a04728e4591d0eede8ee8b712d83489e37470463/pkg/chain/ethereum/beacon.go#L517-L520

Should we remove it as well @lukasz-zimnoch? I do not understand why we need it there. We count on the fact 500 blocks is enough for a DKG to complete now.

Reply:
Yep, we should remove it in that case.

Reply:
Done in 7d3722f79dd2ae45df78a91a92814c7134eb52b0.